internal/platform/platform_linux_test.go

```diff
@@ -0,0 +1,54 @@
+package platform
+
+import (
+	"testing"
+
+	"gotest.tools/v3/assert"
+	"gotest.tools/v3/assert/cmp"
```

Comment:
```suggestion
	is "gotest.tools/v3/assert/cmp"
```

Reply:
Let's be consistent with the rest of the codebase and alias this package to `is`.

Reply:
Sounds good, thanks.